apps/src/tutorialExplorer/filterHeader.jsx

```diff
@@ -69,6 +83,18 @@ const FilterHeader = React.createClass({
           <div style={styles.right}>
             {this.props.filteredTutorialsCount} results
 
+            &nbsp;
+            &nbsp;
```

Comment:
My experience with trying to use &nbsp is that it doesnt always do what I expect (they get stripped in some cases I think). I'd (a) validate that this does what you want it to do and (b) possible replace with something like `{" "}` to force whitespace if it doesnt.